module/os/freebsd/zfs/zfs_vfsops.c

```diff
@@ -2165,7 +2165,7 @@ zfs_vnodes_adjust_back(void)
 #endif
 }
 
-void
+int
```

Comment:
The function needs to actually return something too :)

Reply:
Oh, well. I suppose that might help :). Fixed.